x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/deployment_step.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+// import { useLocation } from 'react-router-dom';
+
+import { ConnectorDeployment } from '../../connector_detail/deployment';
+
+// import { FormattedMessage } from '@kbn/i18n-react';
+
+export const DeploymentStep: React.FC = () => {
+  return <ConnectorDeployment />;
```

Comment:
I tried to consume here your component directly, but it throws an error `Cannot access 'selectDeploymentMethod' before initialization`. Could you @efegurkan try to make it works here?

Reply:
That's a bug. To fix you can move the function in line 82  https://github.com/elastic/kibana/blob/3a2c3f2c75f3883387c9282079d6e6741811edb8/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/deployment.tsx#L82

to above line 64 before the `useEffect`